README.md

```diff
@@ -39,6 +39,7 @@ Creates new BrowserStack workers and returns an array of workers ids.
 		* `browser`: The browser name
 		* `browser_version`: The browser version
 		* `device`: The device name
+		* `worker_name`: The worker name
```

Comment:
Let's write in this way: "The worker name that will be used by TestCafe to identify the browser for test execution."